### PR TITLE
Updating README.md to include Host-Assisted cloning

### LINKED ISSUE #244: document the cdi cloner
Initially the README file can be updated.
@zvikorn

### PR DESCRIPTION
**Pull request by zvikorn**
pulled from code from upstream, fixed three conflicts.

fixes #244

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,6 @@
...
+In Addition, the Containerized Data Importer gives the option to clone the imported VM image from one PVC to another one across two different namespaces (A.K.A Host-Assisted cloning). This is achieved by creating a new PVC with the 'k8s.io/CloneRequest' annotation indicating the name of the PVC the image is copied from. Once the controller detects the PVC, it starts two pods which are responsible for the cloning of the image from one PVC to another using a unix socket that is created on the host itself. When the cloning is completed, the PVC which the image was copied to, is assigned with the 'k8s.io/CloneOf' annotation to indicate cloning completion. The copied VM image can be used by a new pod only after the cloning process is completed.
```
Comment (jeffvance) [inline README.md]: The sentence beginning with "_This is achieved by _" and related details of cloning should be removed from the intro paragraph and moved to the readme section that provides cloning details.

Comment (jeffvance) [reply README.md]: I think Host assisted cloning should be a link in the TOC.

File: README.md [hunk 2]
```diff
@@ -186,6 +187,33 @@ In other words, if the endpoint is a _qcow2_ file, or any file wrapped by tar (e
 
 An additional caveat is that currently the size is under reported for simple iso files, meaning a structured iso file that is not archived and/or compressed.
 
+### Start Cloning Images
+
+Verify the following file from the [example manifests](./manifests/example) exists:
```
Comment (jeffvance) [inline README.md]: If the target pvc manifest does not exist what is the user supposed to do?

File: README.md [hunk 3]
```diff
@@ -186,6 +187,33 @@ In other words, if the endpoint is a _qcow2_ file, or any file wrapped by tar (e
 
 An additional caveat is that currently the size is under reported for simple iso files, meaning a structured iso file that is not archived and/or compressed.
 
+### Start Cloning Images
```
Comment (jeffvance) [inline README.md]: This seems out of place in the doc. It should be before _Endpoint Size_ (which also seems a bit out of place, I was the author of that section). There should be a work-flow order to the _Deploy the API Objects_ section.

File: README.md (hunk 1 above)
Comment (zvikorn) [reply README.md]: ok

File: README.md (hunk 2 above)
Comment (zvikorn) [reply README.md]: changed.

File: README.md (hunk 3 above)
Comment (zvikorn) [reply README.md]: moved.

**Pull request merged**